go/libraries/doltcore/ref/stats_ref.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2023 Dolthub, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ref
+
+// StatsRefName is a dummy name, and there cannot be more than one stash ref.
+const StatsRefName = "stats"
+
+type StatsRef struct {
+	stash string
+}
+
+var _ DoltRef = StatsRef{}
+
+// NewStatsRef creates a reference to a stashes list. There cannot be more than one statsRef.
```

Comment:
At first I thought that "stash" was a typo of "stats" but it's consistent enough that now I'm not sure. Can you clarify why we're calling these "stashes"?

Reply:
sorry bad copy paste, will fix